student.rb

```diff
@@ -9,4 +11,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
   def paly_hooky
     "¯\(ツ)/¯"
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
@andersonlebon, and @AshakaE, there is nothing wrong with this implementation. This is a classic way of creating setters in ruby classes. The `=` signifies the setter method. This is the correct implementation of the `'belongs-to' side (a student belongs to a classroom).`

So @Gedewon, all is good here 👍🏼.